instance/static/html/instance/index.html

```diff
@@ -24,6 +24,13 @@
       <li ng-repeat="instance in instanceList"
           ng-class="{active: instance == selected.instance}">
         <a ng-click="select('instance', instance)">{{ instance.name }}</a>
+        <div class="status-icon" ng-switch="instance.status">
+          <i class="fa fa-ellipsis-h" tooltip-placement="top" tooltip="Provisioning in progress" ng-switch-default></i>
+        </div>
```

Comment:
Sure, there's a problem with that approach as described there. If we want to avoid that problem here as well, we would need to explicitly check whether the status is provisioning and the progress is running, then display this icon.